modules/mapbox/src/deck-utils.ts

```diff
@@ -175,6 +181,26 @@ export function drawLayer(deck: Deck, map: Map, layer: MapboxLayer<any>): void {
   });
 }
 
+function getProjection(map: Map): 'mercator' | 'globe' {
+  const projection = map.getProjection?.();
+  const type =
+    // maplibre projection spec
+    projection?.type ||
+    // mapbox projection spec
+    projection?.name;
+  if (type === 'globe') {
+    return 'globe';
+  }
+  return 'mercator';
+}
+
+export function getDefaultView(map: Map): GlobeView | MapView {
+  if (getProjection(map) === 'globe') {
+    return new GlobeView({id: 'mapbox'});
```

Comment:
Detect maplibre and instead call view `'maplibre'`? Or choose a common name like `'map-overlay'`?

Reply:
We don't have a reliable way to detect mapbox vs maplibre, given that we also support a wide range of versions of each library.

Reply:
I agree it's not the best name. Unfortunately, this is used in some edge cases... @chrisgervang

Reply:
Is it a question of whether this would be a breaking change for the multi-view edge cases?